_posts/0000-01-02-Adrianndenise.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
"You can determine what your future holds based on how much time and energy you spend working on yourself now. Find out what it is you want, and go after it as if your life depends on it. Why? Because it does."